MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/AppearancePatch.cs

```diff
@@ -6,9 +6,13 @@ namespace MixedRealityExtension.Patching.Types
 {
     public class AppearancePatch
     {
-        [PatchProperty]
+        [JsonConverter(typeof(UnsignedConverter))]
         public UInt32? Enabled { get; set; }
```

Comment:
>Enabled [](start = 23, length = 7)

What's this field for?

Reply:
I've commented it in a followup commit. It contains the same data as EnabledPacked, but it's sent on actor creation, and EnabledPacked is sent on update. This is because we don't have a mechanism to convert user-produced ActorLike objects into a different struct for transmission, and I prefer a clean API to clean transmission data.